### ISSUE TITLE
Request: Use Videojs Playback Speed

### ISSUE THREAD

**Issue by eiennosihaisya** [collab-project/videojs-wavesurfer#43]
Videojs has an option for playback speed, like `playbackRates = [0.5, 0.75, 1]`. This adds a control to select the rate. Could this be integrated to change wavesurfer playback rate? (Also, videojs.wavesurfer.css causes this control to overlap the fullscreen button.)

**Comment by thijstriemstra** [comment]
@eiennosihaisya I added support in the master branch, can you try it out?

**Comment by eiennosihaisya** [comment]
Yes, the rates now change wavesurfer playback. However, if a video is not initialized with the wavesurfer plugin, choosing a different playback rate results in the following error with no change:

```
VIDEOJS: ERROR: TypeError: Cannot read property 'surfer' of undefined
    at WavesurferTech.setPlaybackRate (videojs.wavesurfer.js:191)
```

**Comment by thijstriemstra** [comment]
@eiennosihaisya I pushed another fix to master, can you test it?

**Comment by eiennosihaisya** [comment]
Tested and it works! Thank you for the quick response

**Comment by thijstriemstra** [comment]
Will be included in v2.1.1.
